### PR TITLE
[HWORKS-486] remove fk project -> inodes

### PR DESCRIPTION
**Pull request by o-alex**

### CODE REVIEW

File: include/tables/ProjectTable.h [hunk 1]
```diff
@@ -115,28 +113,24 @@ class ProjectTable : public DBTable<ProjectRow> {
...
     ProjectRow row;
     row.mId = values[0]->int32_value();
-    row.mInodeParentId = values[1]->int64_value();
-    row.mInodePartitionId = values[2]->int64_value();
-    row.mInodeName = get_string(values[3]);
-    row.mUserName = get_string(values[4]);
-    row.mDescription = get_string(values[5]);
+    row.mProjectName = get_string(values[1]);
+    row.mUserName = get_string(values[2]);
+    row.mDescription = get_string(values[2]);
```
Comment (maismail) [inline include/tables/ProjectTable.h:133]:
```suggestion
    row.mDescription = get_string(values[3]);
```

File: src/Reindexer.cpp [hunk 2]
```diff
@@ -81,15 +81,16 @@ void Reindexer::run() {
   projectsTable.getAll(metaConn);
   while (projectsTable.next()) {
     ProjectRow project = projectsTable.currRow();
-    INodeRow projectInode = inodesTable.get(conn, project.mInodeParentId,
-            project.mInodeName, project.mInodePartitionId);
+    INodeRow projectsInode = inodesTable.getProjectsInode(conn);
```
Comment (maismail) [inline src/Reindexer.cpp:84]: should be done outside the loop since that won't change

File: include/tables/INodeTable.h [hunk 3]
```diff
@@ -50,9 +50,9 @@ struct INodeRow {
     return mNumUserXAttrs > 0 ||  mNumSysXAttrs > 0;
   }
 
-  bool is_equal(ProjectRow proj){
-    return proj.mInodeName == mName && proj.mInodeParentId == mParentId 
-            && proj.mInodePartitionId == mPartitionId;
+  //assumption is project dirs have parentId = partitionId
```
Comment (maismail) [inline include/tables/INodeTable.h:53]: maybe no need for the is_equal here at all and just put the check explicitly in the Reindexer

**Review by maismail — CHANGES_REQUESTED** [review]

File: include/tables/ProjectTable.h (hunk 1 above)
Comment (o-alex) [reply include/tables/ProjectTable.h:133]: done

File: include/tables/INodeTable.h (hunk 3 above)
Comment (o-alex) [reply include/tables/INodeTable.h:53]: done

**Pull request merged**